core/src/test/scala/unit/kafka/log/LogTest.scala

```diff
@@ -505,7 +575,8 @@ class LogTest extends JUnitSuite {
   @Test
   def testMessageSizeCheck() {
     val first = MemoryRecords.withRecords(CompressionType.NONE, Record.create("You".getBytes), Record.create("bethe".getBytes))
-    val second = MemoryRecords.withRecords(CompressionType.NONE, Record.create("change (I need more bytes)".getBytes))
+    val second = MemoryRecords.withRecords(CompressionType.NONE,
```

Comment:
Maybe worth adding a JIRA or note somewhere to go back and verify this is right later.